ush/set_up_verif_global.sh

```diff
@@ -108,12 +108,20 @@ if [ $machine = "WCOSS2" ]; then
     export FIXverif_global="/lfs/h2/emc/global/noscrub/emc.global/FIX/fix/verif/20220805"
 elif [ $machine = "HERA" ]; then
     export FIXverif_global="/scratch1/NCEPDEV/global/glopara/fix/verif/20220805"
+elif [ $machine = "GAEAC5" ]; then
+    export FIXverif_global="/gpfs/f5/ufs-ard/world-shared/global/glopara/fix/verif/20220805"
+elif [ $machine = "GAEAC6" ]; then
+    export FIXverif_global="/gpfs/f6/bil-fire8/world-shared/global/glopara/fix/verif/20220805"
 elif [ $machine = "ORION" -o $machine = "HERCULES" ]; then
     export FIXverif_global="/work/noaa/global/glopara/fix/verif/20220805"
 elif [ $machine = "S4" ]; then
     export FIXverif_global="/data/prod/glopara/fix/verif/20220805"
 elif [ $machine = "JET" ]; then
     export FIXverif_global="/lfs4/HFIP/hfv3gfs/glopara/git/fv3gfs/fix/verif/20220805"
+elif [ $machine = "GAEAC5" ]; then
+    export FIXverif_global="/gpfs/f5/ufs-ard/world-shared/global/glopara/fix/verif/20220805"
+elif [ $machine = "GAEAC6" ]; then
+    export FIXverif_global="/gpfs/f6/bil-fire8/world-shared/global/glopara/fix/verif/20220805/"
```

Comment:
We have that fix data under role.glopara on each supported platform, just need to change the Gaea paths to:
```suggestion
elif [ $machine = "GAEAC5" ]; then
    export FIXverif_global="/gpfs/f5/nggps_emc/world-shared/role.glopara/DATA/FIX/verif/20220805"
elif [ $machine = "GAEAC6" ]; then
    export FIXverif_global="/gpfs/f6/drsa-precip3/world-shared/role.glopara/fix/verif/20220805/"
```